uint16 model_num
uint8 selector
uint32 flags
uint16 sys_state
uint32 sys_timer 
